src/__tests__/resetpassword/ForgotPassword.test.jsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import Enzyme, { shallow } from "enzyme";
+import moxios from "moxios";
+import Adapter from "enzyme-adapter-react-16";
+import configureMockStore from "redux-mock-store";
+import thunk from "redux-thunk";
+import { ForgotPassword } from "../../components/ForgotPasswordComp";
+import { FORGOT_PWD } from "../../redux/action_types";
+import { forgotPassword } from "../../redux/actions/ForgotPasswordAction";
+
+Enzyme.configure({ adapter: new Adapter() });
+const middlewares = [thunk];
+const mockStore = configureMockStore(middlewares);
+describe("user action", () => {
+  beforeEach(() => {
+    moxios.install();
+  });
+  afterEach(() => {
+    moxios.uninstall();
+  });
+  const email = "test@gmail.com";
+  it("should dispatch an action to generate email address", () => {
+    const expectedAction = [
+      {
+        type: FORGOT_PWD,
+        responseData: "test@gmail.com"
+      }
+    ];
+    //   configure Mock store
+    const store = mockStore({ response: { message: "check your email" } });
```

Comment:
@meshack-mbuvi I suggest you declare the store globally and then clear actions in the `afterEach()` function

Reply:
Your feedback has already been taken into account.